src/program/views.py

```diff
@@ -771,10 +781,19 @@ class EventListView(CampViewMixin, ListView):
     model = models.Event
     template_name = "event_list.html"
 
+    def get_queryset(self, *args, **kwargs):
+        qs = super().get_queryset(*args, **kwargs)
+        qs = qs.prefetch_related("event_type")
+        qs = qs.prefetch_related("track")
+        qs = qs.prefetch_related("instances")
+        qs = qs.prefetch_related("speakers")
```

Comment:
Couldn't this be one `prefetch_related` call?

Reply:
yes, I think they were like this because it made it easy to test which combinations helped and which didn't. Fixed in 9c5341e364b88edfe938184b78d6bb398b356ae8